x-pack/plugins/siem/public/components/navigation/index.tsx

```diff
@@ -10,7 +10,7 @@ import { RouteComponentProps, withRouter } from 'react-router-dom';
 import { setBreadcrumbs } from './breadcrumbs';
 import { TabNavigation } from './tab_navigation';
 
-export class SiemNavigationComponent extends React.Component<RouteComponentProps> {
+export class SiemNavigationComponent extends React.PureComponent<RouteComponentProps> {
```

Comment:
It complains about using `shouldComponentUpdate ` in React.PureComponent, would we want to keep this as React.Component in this case?